ledokku-bootstrap.sh

```diff
@@ -69,15 +208,15 @@ main() {
   dokku config:set ledokku JWT_SECRET="${JWT_SECRET}"
 
   # Now it's finally time to install ledokku
-  echo "=> Installation"
+  echo "=> ${YELLOW}Installing ledooku${END}"
   dokku git:from-image ledokku ledokku/ledokku:${LEDOKKU_TAG}
```

Comment:
Dokku has introduced the `git:from-image <app> <docker-image>` from **dokku v0.24.0** 

The digital ocean marketplace provide a **dokku v0.21.4** => we should force users to have a minimum of dokku v0.24.0

Reply:
fixed: https://github.com/ledokku/ledokku/pull/393/commits/ac2b0e3906a578a7bcc991683e4562d5c45deb98